InventoryDistanceBasedSourceSelection/Console/Command/ImportGeoNamesCommand.php

```diff
@@ -22,23 +25,33 @@ class ImportGeoNamesCommand extends Command
 {
     private const COUNTRIES = 'countries'; // Parameter name for countries list
 
+    private const URL = 'url'; //Parameter name for download countries url.
+
```

Comment:
Instead of adding comment inline, just rename const to be meaningful: `PARAM_...` eg. `PARAM_COUNTRIES_URL`